app/assets/javascripts/components.js

```diff
@@ -0,0 +1,36 @@
+//= require_self
+//= require react_ujs
+
+var React = require('react/addons');
+
+// put react components here
+
+var InfiniteScroll = require('./react_components/react-infinite-scroll');
+var ReactCSSTransitionGroup = React.addons.CSSTransitionGroup;
+
+/* Book Search Page */
+var BookList = require('./react_components/book');
+var TagModalContent = require('./react_components/tag_modal');
+var BookTile = require('./react_components/book_tiles');
+
+/* Users BookLists */
+var PublishedBookLists = require('./react_components/book_lists');
+var BookLists = require('./react_components/users/booklists');
+var BookListTable = require('./react_components/users/booklist_table');
+
+/* Admin Dashboard */
+var DashboardTabs = require('./react_components/admin_dashboard/admin_dashboard');
+var BookStatusView = require('./react_components/admin_dashboard/book_statuses');
+
+/* Admin Dasboard BaseLists */
+
+var BaseListView = require('./react_components/admin_dashboard/baselists/baselist_view'); var BaseLists = require('./react_components/admin_dashboard/baselists/baselists');
```

Comment:
Line is too long.<br>'BaseListView' is defined but never used.<br>'BaseLists' is defined but never used.